Makefile

```diff
@@ -325,6 +336,7 @@ build-pytorch-cuda: build-cuda-118-base
 		-t $(DOCKERHUB_REGISTRY)/$(CUDA_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH) \
 		-t $(NGC_REGISTRY)/$(CUDA_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH) \
 		.
+	docker run --rm -v `pwd`/tests:/workspace/tests -it $(DOCKERHUB_REGISTRY)/$(CUDA_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH) /bin/bash -c "pip install pytest && pytest -m pytorch /workspace/tests"
 
 .PHONY: publish-tensorflow-cpu
 publish-tensorflow-cpu:
```

Comment:
Should have a test here as well

Reply:
You mean a test for `publish-tensorflow-cpu`?